src/command_modules/azure-cli-vsts/azure/cli/command_modules/vsts/custom.py

```diff
@@ -192,49 +197,66 @@ def list_releases(name, resource_group_name):
     json_request = req.json()
     return json_request
 
+def _gitroot():
+    """
+    Gets the absolute path of the repository root
+    """
+    try:
+        base = check_output(['git', 'rev-parse', '--show-toplevel'])
+    except OSError:
+        raise CLIError('Git is not currently installed.')
+    except CalledProcessError:
```

Comment:
Need to add `CalledProcessError` to the `from subprocess import ...` line at the very top.